static/scss/faculty-members.scss

```diff
@@ -0,0 +1,111 @@
+// sass-lint:disable mixins-before-declarations
+.faculty-section {
+  margin: 0 0 20px;
+
+  .container {
+    h2 {
+      margin: 0 0 35px;
+
+      @include media-breakpoint-down(sm) {
+        margin: 0 0 27px;
+      }
+    }
+  }
+}
+
+.faculty-members {
+  padding: 0;
+  margin: 0 -22px;
+  list-style: none;
+  letter-spacing: -0.32em;
+
+  .course-card-description {
+    overflow-y: auto;
+    margin-bottom: 30px;
+  }
+
+  li {
+    @include media-breakpoint-down(md) {
+      width: calc(100% - 44px);
+    }
+
+    letter-spacing: normal;
+    display: inline-block;
+    vertical-align: top;
+    width: calc(50% - 44px);
+    margin: 0 22px 24px;
+
+    .member-card {
+      padding: 24px;
+      background: white;
+      border-bottom: 3px solid transparent;
+      box-shadow: 0 2px 4px 0 rgba(0,0,0,0.16);
+      overflow: hidden;
+
+      .member-info {
+        overflow: hidden;
+        margin-top: 7px;
+
+        @include media-breakpoint-down(md) {
+          margin-top: 20px;
+        }
+
+        h2 {
+          color: #000000;
+          font-size: 18px;
+          font-weight: 600;
+          letter-spacing: 0;
+          line-height: 22px;
+          margin: 0 0 5px;
+        }
+
+        p {
+          color: #333333;
+          font-size: 14px;
+          font-weight: 400;
+          letter-spacing: 0;
+          line-height: 22px;
+          margin-bottom: 0;
+        }
+
+        .designation {
+          p {
+            margin: 0;
+            font-weight: 500;
+          }
+        }
+
+        .description {
+          p {
+            margin: 0;
+            color: #767676;
+          }
+        }
+      }
+
+      &:hover {
+        border-color: #a41e34;
+        box-shadow: 0 2px 15px 4px rgba(0, 0, 0, 0.16);
+
+        h2 {
+          color: #a41e34;
```

Comment:
`#a41e34` is defined twice here, and the `box-shadow` styling is basically the same as the course cards on the home page. That makes sense, of course. In both cases, they are cards that have a box shadow and a colored bottom border when you hover over them. Obviously we do not want to define the same set of styles for elements that have the exact same look and feel. 

Please define a class in `common.scss` or somewhere like that, call it something like `highlight-card`, apply it to both the home page cards and these faculty cards, and get rid of the repeated styles. Like I said above, we need to make a greater effort to reuse existing styles and reduce the amount of styling code we're adding to the codebase